### PR TITLE
Bring fix images private sites solution to release 12 1

### PR DESCRIPTION
**Pull request by mzorz**
Brings the fix in https://github.com/wordpress-mobile/WordPress-Android/pull/9523 to `release/12.1`

To test:
- Follow  steps described in #9523 

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by mzorz** [comment]
@jtreanor @elibud made this PR in case the fix made by @marecar3 needs to get into `release/12.1` before Monday - I reviewed and merged [the original PR](https://github.com/wordpress-mobile/WordPress-Android/pull/9523) to `develop` already so it's safely into code freeze 👍

**Comment by elibud** [comment]
I think it would be great to ship this on Monday if we feel confident about the fix.

**Comment by marecar3** [comment]
Hey @mzorz we can merge this one! Just there is one comment.

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java [hunk 1]
```diff
@@ -41,6 +47,13 @@ public static EditPostPreviewFragment newInstance(@NonNull PostModel post) {
     public void setArguments(Bundle args) {
         super.setArguments(args);
         mLocalPostId = args.getInt(ARG_LOCAL_POST_ID);
+        mSite = (SiteModel) args.getSerializable(WordPress.SITE);
+    }
+
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
```
Comment (marecar3) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java]: My latest commit on the PR doesn't have this method. https://github.com/wordpress-mobile/WordPress-Android/pull/9523/files

Comment (mzorz) [reply WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java]: thanks for the heads up - addressed in 7cf3d94bb1. Will merge after Travis goes green.

**Pull request merged**